posthog/migrations/0061_featureflag.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.7 on 2020-06-17 08:33
+
+from django.conf import settings
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('posthog', '0060_auto_20200616_0746'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FeatureFlag',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
How could we track the usage of feature flags? I think it's important to find  out how often value of the decision is given so you have the analyticss.

Reply:
In the posthog-js array we pass a `$active_feature_flag` property with an array of the feature flags that were active at that point. That way we can do all the filtering we want using the /trends endpoint for example

Reply:
Yes, can we this way track how often `false` is given and how often `true` is given? I think that's quite a useful metric to keep track of